dev/snippets/config/templates/stateful_widget.tmpl

```diff
@@ -10,12 +10,13 @@ class MyApp extends StatelessWidget {
   // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
-    return new MaterialApp(
+    return MaterialApp(
       title: 'Flutter Code Sample for {{id}}',
-      theme: new ThemeData(
+      theme: ThemeData(
         primarySwatch: Colors.blue,
       ),
-      home: new MyStatefulWidget(),
+      home: Material(child: MyStatefulWidget()),
+      color: Colors.white,
```

Comment:
It would help to explain why we're specifying this. If the app includes a Scaffold it's not needed.